test/integration/spec/localparticipant.js

```diff
@@ -364,6 +365,63 @@ const isSafari = guess === 'safari';
         });
       });
     });
+
+    describe('"trackPublicationFailed" event', () => {
+      combinationContext([
+        [
+          [
+            {
+              createLocalTrack() {
+                const { name } = tracks.find(track => track.kind === 'audio');
+                console.log('Name:', name);
+                return createLocalAudioTrack({ name });
+              },
+              scenario: 'a LocalTrack whose name is the same as one of the currently published LocalTracks',
+              TwilioError: TrackNameIsDuplicatedError
+            },
+            {
+              createLocalTrack() {
+                const name = '0'.repeat(130);
```

Comment:
Cool, didn't know about `repeat`. So the max length is what? 129?

Reply:
@markandrus 128 characters. I could change the above number to 129.